Source/DafnyLanguageServer.Test/Util/Stringify.cs

```diff
@@ -9,9 +9,20 @@
 
 namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Util;
 
+using Overrides = ImmutableDictionary<System.Type, Action<object, TextWriter>>;
 public static class StringifyUtil {
 
-  public static void Stringify(this object root, TextWriter writer, bool showNullChildren = false) {
+  public static Overrides EmptyOverrides() {
+    return ImmutableDictionary<System.Type, Action<object, TextWriter>>.Empty;
+  }
+
+  public static Overrides UseToString(this Overrides overrides, System.Type type) {
+    return overrides.Add(type, (value, writer) => writer.Write(value.ToString()));
+  }
+
+  public static void Stringify(this object root, TextWriter writer, bool showNullChildren = false, IReadOnlyDictionary<System.Type, Action<object, TextWriter>> overrides = null) {
```

Comment:
Nullable checks are enabled in this file, so there's a warning about `overrides` not being nullable.